actors/miner/src/monies.rs

```diff
@@ -173,42 +185,32 @@ pub fn pledge_penalty_for_termination_lower_bound(
     )
 }
 
-/// Penalty to locked pledge collateral for the termination of a sector before scheduled expiry.
-/// SectorAge is the time between the sector's activation and termination.
-#[allow(clippy::too_many_arguments)]
+/// Calculates termination fee for a given sector. Normally, it's calculated as a fixed percentage
+/// of the initial pledge. However, there are some special cases outlined in the
+/// [FIP-0098](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0098.md).
 pub fn pledge_penalty_for_termination(
-    day_reward: &TokenAmount,
+    initial_pledge: &TokenAmount,
     sector_age: ChainEpoch,
-    twenty_day_reward_at_activation: &TokenAmount,
-    network_qa_power_estimate: &FilterEstimate,
-    qa_sector_power: &StoragePower,
-    reward_estimate: &FilterEstimate,
-    replaced_day_reward: &TokenAmount,
-    replaced_sector_age: ChainEpoch,
+    fault_fee: &TokenAmount,
 ) -> TokenAmount {
-    // max(SP(t), BR(StartEpoch, 20d) + BR(StartEpoch, 1d) * terminationRewardFactor * min(SectorAgeInDays, 140))
-    // and sectorAgeInDays = sectorAge / EpochsInDay
-    let lifetime_cap = TERMINATION_LIFETIME_CAP * EPOCHS_IN_DAY;
-    let capped_sector_age = std::cmp::min(sector_age, lifetime_cap);
-
-    let mut expected_reward: TokenAmount = day_reward * capped_sector_age;
-
-    let relevant_replaced_age =
-        std::cmp::min(replaced_sector_age, lifetime_cap - capped_sector_age);
-
-    expected_reward += replaced_day_reward * relevant_replaced_age;
+    // Use the _percentage of the initial pledge_ strategy to determine the termination fee.
+    let simple_termination_fee =
+        (initial_pledge * TERM_FEE_PLEDGE_MULTIPLE_NUM).div_floor(TERM_FEE_PLEDGE_MULTIPLE_DENOM);
+
+    // Apply the age adjustment for young sectors to arrive at the base termination fee.
+    let base_termination_fee = cmp::min(
+        simple_termination_fee.clone(),
+        (sector_age * &simple_termination_fee).div_floor(TERMINATION_LIFETIME_CAP * EPOCHS_IN_DAY),
+    );
```

Comment:
uber-nit (saves a clone, but feel free to ignore):

```suggestion
    let duration_termination_fee =
        (sector_age * &simple_termination_fee).div_floor(TERMINATION_LIFETIME_CAP * EPOCHS_IN_DAY);

    // Apply the age adjustment for young sectors to arrive at the base termination fee.
    let base_termination_fee = cmp::min(simple_termination_fee, duration_termination_fee);
```